labs/lab-1-html-and-css/index.html

```diff
@@ -1,45 +1,51 @@
 <!DOCTYPE html>
 <html>
 
-  <head>
+<head>
     <title>Chirpus Inc.</title>
     <link href="mobilefirststyle.css" rel="stylesheet" type="text/css">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
-  </head>
-  <body>
-    
-      <h3>chirpus inc.</h3>
-     <div class ="links">
-         <a href="html_images.asp">Home</a>
-         <a href="html_images.asp">Work</a>
-         <a href="html_images.asp">About us </a>
-         <a href="html_images.asp">Contact</a>
-     </div>
-    <h2 id = "hireButton">Hire Us</h2>
-    
+</head>
+
+<body>
+
+    <h3 class="logo">chirpus inc.</h3>
+    <nav>
+        <div class="menu">
+            <ul>
+                <div class="nav-link">
+                    <a href="html_images.asp">Home</a>
+                    <a href="html_images.asp">Work</a>
+                    <a href="html_images.asp">About us </a>
+                    <a href="html_images.asp">Contact</a>
+                </div>
+                <nav id="hireButton">Hire Us</nav>
```

Comment:
`nav` is an element that should contain the whole navbar; I'd use another `<a>` here.

If you use an `<a>` for "Hire Us" and put it *inside* the `nav-link` `div` instead of outside it, then it'll be positioned where you wanted it (but you'll need to use `display: inline-block`, I suspect, to make it look perfect).